components/rmrk/Gallery/CollectionItem.vue

```diff
@@ -182,21 +182,23 @@ const components = {
 export default class CollectionItem extends mixins(
   ChainMixin,
   PrefixMixin,
-  CreatedAtMixin
+  CreatedAtMixin,
+  InfiniteScrollMixin
 ) {
   @Ref('tabsContainer') readonly tabsContainer
   private id = ''
   private collection: CollectionWithMeta = emptyObject<CollectionWithMeta>()
   public meta: CollectionMetadata = emptyObject<CollectionMetadata>()
-  private searchQuery: SearchQuery = {
-    search: '',
-    type: '',
-    sortBy: 'BLOCK_NUMBER_DESC',
-    listed: false,
-  }
+  private searchQuery: SearchQuery = Object.assign(
+    {
+      search: '',
+      type: '',
+      sortBy: (this.$route.query.sort as string) ?? 'blockNumber_DESC',
```

Comment:
Here is your bug [2862](https://github.com/kodadot/nft-gallery/issues/2862)

Reply:
Yes, it is : )